### DISCUSSION TITLE
Error when running the new backup script

### DISCUSSION THREAD

**Discussion by josvink66** [amidaware/tacticalrmm#1564] [General]
**Server Info (please complete the following information):**
 - OS:Ubuntu 20.04
 - Browser: n/a
 - RMM Version v0.15.12

**Installation Method:**
  - [X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): n/a
- Agent OS: n/a

**Describe the bug**
When running the new backup.sh script it throw an error (see screenhot). The backup process does continue but in the end the file necessary for the mongo to postgresql conversion is not created. So I can't use the backup for the restore on the new OS,

To be absolute sure  I was using the latest backup.sh I made sure to remove the old version.

**To Reproduce**
wget -N https://raw.githubusercontent.com/amidaware/tacticalrmm/master/backup.sh
chmod +x backup.sh
./backup.sh

**Expected behavior**
A full backup file with the file necessary for the mongo to postgresql conversion.

**Screenshots**
![image](https://github.com/amidaware/tacticalrmm/assets/21361769/5046ccad-358a-4666-bb33-df98bab6c319)

**Additional context**
Running the command that throws the error manually gives the same error:
![image](https://github.com/amidaware/tacticalrmm/assets/21361769/438df9c2-dc32-4a2a-b7d0-13edcc569839)

**Comment by dinger1986** [comment]
I just ran `node /meshcentral/node_modules/meshcentral --dbexport` on my trmm server and it worked fine.

Do you have anything else on that server?

**Comment by josvink66** [comment]
Hi @dinger1986   thanks for the quick response. No this server is only for RMM. There have never been anything else on it.

**Comment by dinger1986** [comment]
can you do `ls -l /meshcentral` and post the output

**Comment by josvink66** [comment]
Yes! here it is:

![image](https://github.com/amidaware/tacticalrmm/assets/21361769/b4c4ba0c-34ec-4932-a786-18ab6837e6df)

**Comment by wh1te909** [comment]
can you please run the following (as the `tactical` user) and then try the backup again. let me know if any of the commands below error out. this will kill any connections to meshcentral so do it during off hours if needed.

```bash
sudo systemctl stop meshcentral
sudo chown tactical:tactical -R /meshcentral
cd /meshcentral
rm -rf node_modules/
npm install meshcentral@1.1.6
sudo systemctl start meshcentral
```

**Comment by josvink66** [comment]
@wh1te909 That seems to have worked no errors. Now i'm going to test the restore. Thanks!
